elliottlib/imagecfg.py

```diff
@@ -23,32 +24,25 @@ def base_only(self):
         """
         return self.config.base_only
 
-    def get_latest_build_info(self, product_version):
+    def get_latest_build_info(self, product_version, tag_set, latest_builds, brew_session):
```

Comment:
This function also returns product version in the tuple. This doesn't match the comment.
Also `product_version` sounds like a single string, but it is actually a dict. The structure and purpose of the dict is not documented.

Reply:
sure, will add later